docs/index.md

```diff
@@ -460,6 +460,35 @@ async def app(scope, receive, send):
     })
 ```
 
+## The WSGI Support
+
+Uvicorn offer support to WSGI applications throught `WSGIMiddleware` which converts
+your WSGI into ASGI application seamlessly just by running it with `--interface wsgi`
+
+!!! warning
+    The present WSGIMiddleware will be deprecated in favour of [a2wsgi](https://github.com/abersheeran/a2wsgi).
+    If you are using Uvicorn to run WSGI applications, please add it as part of your project requirements.
```

Comment:
It will be **removed**, but this PR already deprecates it.
```suggestion
    The present WSGIMiddleware is deprecated in favour of [a2wsgi](https://github.com/abersheeran/a2wsgi).
    If you are using Uvicorn to run WSGI applications, please add it as part of your project requirements.
```

Reply:
Yup, updated the docs